modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -618,65 +621,92 @@ The output represents the transformed XML elements into JSON.
 ]
 ----
 
+// CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
 
-// CONFIG PROPS ///////////////////////////////////////////////////////
 
 [[properties]]
 == Configuration Properties
 
-DataWeave supports the following configuration properties for the XML format.
+DataWeave supports the following configuration properties for this format.
+
 
+[[reader_properties]]
 === Reader Properties
 
-The XML format accepts properties that provide instructions for reading input data.
+This format accepts properties that provide instructions for reading input data.
+
 
 [cols="1,1,1,3a", options="header"]
 |===
-| Parameter | Type | Default | Description
-|`collectionPath`  |`String`| `null` | Path to the document where the collection is located. It specifies a path expression that identifies the location of the elements to stream. See <<example1>>.
-| `externalEntities` | `Boolean` | `false` | Indicates whether external entities
-should be processed or not. By default this is disabled to avoid XXE attacks.
-Valid values are `true` or `false`.
-| `indexedReader` | `Boolean` | `true` | Indicates whether to use the indexed XML reader when the threshold is reached. Valid values are `true` or `false`.
-| `maxAttributeSize` | `Number` | `-1` | Sets the maximum number of characters
-accepted in an XML attribute. Available since Mule 4.2.1.
-| `maxEntityCount` | `Number` | `1` | The maximum number of entity expansions.
-The limit is in place to avoid Billion Laughs attacks.
-| `nullValueOn` | `String` | `blank` | Indicates whether to read an element with empty or blank text as a `null` value. Valid values are `empty`, `none`, or `blank`. See <<example3>> and <<example4>>.
-|`optimizeFor` |`String`| `'speed'` | Indicates how to configure the XML parser. Valid values are `speed` and `memory`.
-|`streaming` |`Boolean` | `false` | Indicates whether to stream input (use only if entries are accessed sequentially). Valid Options are `true` or `false`. See <<example1>>.
-|`supportDtd` |`Boolean` | `false` |  Enable or disable DTD support. Disabling skips (and does not process) internal and external subsets. Valid values are `true` or `false`. You can also enable this property by setting the Mule system property `com.mulesoft.dw.xml.supportDTD`. Note that the default for this property changed from `true` to `false` in Mule version 4.3.0-20210601, which includes the June 2021 patch of DataWeave version 2.3.0.
+|Parameter |Type |Default|Description
+|`collectionPath`  |`String`|`null`|Sets the path to the location in the document where the collection is located.
+|`externalEntities`  |`Boolean`|`false`|Indicates whether to process external entities. Disabled by default to avoid XML External Entity (XXE) attacks. Valid values are `true` or `false`.
+ Valid values are `true` or `false`.
+|`indexedReader`  |`Boolean`|`true`|If the indexed reader should be used when the threshold is reached.
```

Comment:
TODO: COPYEDIT
"Indicates whether to use the indexed XML reader when the threshold is reached. Valid values are `true` or `false`."